corral/bundle/bundle.pony

```diff
@@ -34,7 +33,8 @@ primitive BundleFile
         Error("Error loading bundle files in " + dir.path)
       end
     else
-      Error("No " + Files.bundle_filename() + " in current working directory or ancestors.")
+      Error("No " + Files.bundle_filename() +
+        " in current working directory or ancestors.")
```

Comment:
Binary operators should come prior to the expression on the next line: https://github.com/ponylang/ponyc/blob/master/STYLE_GUIDE.md#assignment